pkgs/by-name/sy/sysmenu/package.nix

```diff
@@ -0,0 +1,55 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  git,
+  gtkmm4,
+  gtk4-layer-shell,
+  wrapGAppsHook4,
+  unstableGitUpdater,
+}:
+stdenv.mkDerivation {
+  pname = "sysmenu";
+  version = "unstable-13-06-2024";
```

Comment:
```suggestion
  version = "unstable-2024-06-13";
```

Reply:
also commit message

Reply:
@Aleksanaa I'm using that formatting as far as I am aware?

![image](https://github.com/NixOS/nixpkgs/assets/3139724/8afc2e11-2c17-4433-8f8a-a30f4f9fcb27)